src/index.ts

```diff
@@ -144,6 +149,12 @@ export interface User {
   postalCode?: string
   addressLine1?: string
   addressLine2?: string
+
+  // dataSubjectTypeCode is the identifier representing the data subject type specified by the user
+  dataSubjectTypeCode?: string
+
+  // dataSubjectTypeUserInput is additional information provided by the user describing their relation to the business
+  dataSubjectTypeUserInput?: string
```

Comment:
not sure what this means.

Reply:
We want to support data subject types that require more information from the user. 

A use case would be an "Other" data subject type where below that there is a text box for a user to add more information about their relation to the business. This field would send what the user enters in this text box